### ISSUE TITLE
Laravel 8 support

### ISSUE THREAD

**Issue by freebuu** [irazasyed/telegram-bot-sdk#817]
Hello!
Laravel8 contais Guzzle ^7, but it conflicts with requirments:
```
- Root composer.json requires irazasyed/telegram-bot-sdk ^2.3 -> satisfiable by irazasyed/telegram-bot-sdk[v2.3.0].
- irazasyed/telegram-bot-sdk v2.3.0 requires guzzlehttp/guzzle ~6.0 -> found guzzlehttp/guzzle[6.0.0, ..., 6.5.x-dev] but it conflicts with your root composer.json require (^7.0.1).
```

**Comment by Ziya584** [comment]
Is there any solution?

**Comment by roelofr** [comment]
This issue was resolved in dfa43245b7b6071e1d1fc566bd6462cbf549a1bb, updating to `^v3.3` should fix your problem.
